src/Elements/Input.react.js

```diff
@@ -182,6 +183,15 @@ class Input extends Component {
                 this.setState({ inputActionsTopPosition: inputTop });
             }
         }
+
+        if(autoFocus) {
+            let len = this.state.value.length;
+            this.state.value[len-1].focus();
+
+            this.setState({
+                isFocused: true
+            })
+        }
```

Comment:
I'm wondering if this should just be set in the `constructor()` instead of having the component render again?

Reply:
It does help with updating the classes that are on the components.